config/datatypes_conf.xml.sample

```diff
@@ -38,6 +38,7 @@
     <datatype extension="len" type="galaxy.datatypes.chrominfo:ChromInfo" display_in_upload="true">
       <converter file="len_to_linecount.xml" target_datatype="linecount" />
     </datatype>
+    <datatype extension="dmnd" type="galaxy.datatypes.binary:Binary:DMND" subclass="True" display_in_upload="false"/>
```

Comment:
Is there a reason why this shouldn't be uploaded ? Given the sniffer is there we might just flip that.